clientv3/op.go

```diff
@@ -117,18 +117,30 @@ func (op *Op) applyOpts(opts []OpOption) {
 	}
 }
 
+// OpOption configures Op operations.
 type OpOption func(*Op)
 
+// WithLease attaches a lease ID to a key in 'Put' request.
 func WithLease(leaseID lease.LeaseID) OpOption {
 	return func(op *Op) { op.leaseID = leaseID }
 }
+
+// WithLimit limits the number of results to return from 'Get' request.
 func WithLimit(n int64) OpOption { return func(op *Op) { op.limit = n } }
+
+// WithRev specifies the store revision for 'Get' request.
 func WithRev(rev int64) OpOption { return func(op *Op) { op.rev = rev } }
+
+// WithSort specifies the order in 'Get' requests
```

Comment:
WithSort specifies the ordering in 'Get' request.
It requires WithRange and/or WithPrefix to be specified too.